### ISSUE TITLE
模型训练卡着不动

### ISSUE THREAD

**Issue by zhuangzexuan** [PaddlePaddle/PaddleOCR#15169]
### 🔎 Search before asking

- [x] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [x] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [x] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

2025/05/19 10:36:38] ppocr INFO: Initialize indexes of datasets:['train_data/icdar2015/text_localization/train_icdar2015_label.txt']
[2025/05/19 10:36:38] ppocr INFO: Initialize indexes of datasets:['train_data/icdar2015/text_localization/test_icdar2015_label.txt']
INFO:root:If regularizer of a Parameter has been set by 'paddle.ParamAttr' or 'static.WeightNormParamAttr' already. The weight_decay[3e-05] in Optimizer will not take effect, and it will only be applied to other Parameters!
[2025/05/19 10:36:39] ppocr INFO: train dataloader has 188 iters
[2025/05/19 10:36:39] ppocr INFO: valid dataloader has 89 iters
[2025/05/19 10:36:39] ppocr INFO: load pretrain successful from pretrained_models/en_PP-OCRv3_rec_train/en_PP-OCRv3_rec_train/best_accuracy
[2025/05/19 10:36:39] ppocr INFO: During the training process, after the 0th iteration, an evaluation is run every 100 iterations

python tools/train.py -c configs/rec/PP-OCRv3/en_PP-OCRv3_rec.yml
使用上述命令启动训练时，就一直卡在ppocr INFO那一行 也不报错也不继续运行

### 🏃‍♂️ Environment (运行环境)

windows
python                        3.8
paddleocr                    2.10.0
paddlepaddle-gpu             2.6.2

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

Global:
  debug: false
  use_gpu: false
  epoch_num: 100
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/v3_en_mobile
  save_epoch_step: 20
  eval_batch_step: [0, 100]
  cal_metric_during_train: true
  pretrained_model: pretrained_models/en_PP-OCRv3_rec_train/en_PP-OCRv3_rec_train/best_accuracy
  checkpoints:
  save_inference_dir:
  use_visualdl: false
  infer_img: doc/imgs_words/ch/word_1.jpg
  character_dict_path: ppocr/utils/en_dict.txt
  max_text_length: &max_text_length 25
  infer_mode: false
  use_space_char: true
  distributed: true
  save_res_path: ./output/rec/predicts_ppocrv3_en.txt

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.001
    warmup_epoch: 5
  regularizer:
    name: L2
    factor: 3.0e-05

Architecture:
  model_type: rec
  algorithm: SVTR_LCNet
  Transform:
  Backbone:
    name: MobileNetV1Enhance
    scale: 0.5
    last_conv_stride: [1, 2]
    last_pool_type: avg
    last_pool_kernel_size: [2, 2]
  Head:
    name: MultiHead
    head_list:
      - CTCHead:
          Neck:
            name: svtr
            dims: 64
            depth: 2
            hidden_dims: 120
            use_guide: True
          Head:
            fc_decay: 0.00001
      - SARHead:
          enc_dim: 512
          max_text_length: *max_text_length

Loss:
  name: MultiLoss
  loss_config_list:
    - CTCLoss:
    - SARLoss:

PostProcess:  
  name: CTCLabelDecode

Metric:
  name: RecMetric
  main_indicator: acc
  ignore_space: False

Train:
  dataset:
    name: SimpleDataSet
    data_dir: train_data/icdar2015/text_localization
    ext_op_transform_idx: 1
    label_file_list:
    - train_data/icdar2015/text_localization/train_icdar2015_label.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - RecConAug:
        prob: 0.5
        ext_data_num: 2
        image_shape: [48, 320, 3]
        max_text_length: *max_text_length
    - RecAug:
    - MultiLabelEncode:
    - RecResizeImg:
        image_shape: [3, 48, 320]
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_sar
        - length
        - valid_ratio
  loader:
    shuffle: true
    batch_size_per_card: 4
    drop_last: true
    num_workers: 1
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: train_data/icdar2015/text_localization
    label_file_list:
    - train_data/icdar2015/text_localization/test_icdar2015_label.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - MultiLabelEncode:
    - RecResizeImg:
        image_shape: [3, 48, 320]
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_sar
        - length
        - valid_ratio
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 4
    num_workers: 1

**Comment by leo-q8** [comment]
需要自定义一下数据集路径

**Comment by TingquanGao** [comment]
The issue has no response for a long time and will be closed. You can reopen or new another issue if are still confused.

---
_From Bot_
